app/api/v1/dependencies/oauth_email.py

```
@@ -0,0 +1,98 @@
+
+async def get_oauth_provider_config_repo_for_write(
+    db: Annotated[AsyncSession, Depends(get_db_transactional)],
+    audit_svc: Annotated[SystemAuditService, Depends(db_deps.get_system_audit_service)],
+) -> OAuthProviderConfigRepository:
+    """OAuth provider config repository for create/update/delete (transactional)."""
+    return OAuthProviderConfigRepository(db, audit_service=audit_svc)
+
+
+async def get_oauth_state_repo(
+    db: Annotated[AsyncSession, Depends(get_db_transactional)],
+    state_manager: OAuthStateManager = Depends(auth.get_oauth_state_manager),
```

Ruff (0.15.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)